### PR TITLE
fixes for IPDBPath without `.gz`

### PR DESCRIPTION
**Pull request by liut**
example scene:

```go
f, err := New(Options{
	IPDBPath:       "/var/tmp/GeoLite2-Country.mmdb",
	IPDBNoFetch:    true,
	AllowedIPs:     []string{"222.25.118.1"},
	BlockByDefault: true,
})
```

### CODE REVIEW

File: ipfilter.go [hunk 1]
```diff
@@ -175,7 +176,7 @@ func (f *IPFilter) initDB() error {
 	defer resp.Body.Close()
 	//store on disk as db loads
 	r := io.TeeReader(resp.Body, file)
-	err = f.readerDB(DBPublicURL, r)
+	err = f.readerDB(f.opts.IPDBPath, r)
```
Comment (jpillora) [inline ipfilter.go]: This line should be `f.opts.IPDBFetchURL` I think...

File: ipfilter.go [hunk 2]
```diff
@@ -140,13 +140,14 @@ func (f *IPFilter) initDB() error {
 	}
 	//use local copy
 	if fileinfo, err := os.Stat(f.opts.IPDBPath); err == nil {
+		// f.opts.Logger.Printf("[ipfilter] IPDBPath %q Found", f.opts.IPDBPath)
```
Comment (jpillora) [inline ipfilter.go]: Please remove unused prints

File: ipfilter.go (hunk 1 above)
Comment (liut) [reply ipfilter.go]: In `readerDB(filename string, r Reader)`, the first argument just as a FileName for gzReader, not a URL. The Correct Suffix(gz or not gz) of IPDB, decision is developer options.

**Comment by jpillora** [comment]
Thanks for your work, though I've just pushed  a change to remove maxminddb, in favour of iplocation data (see https://github.com/phuslu/geoip)

**Pull request closed without merging**